### ISSUE TITLE
Can no longer switch between GraphQL operations with the mouse cursor

### ISSUE THREAD

**Issue by zleonov** [getinsomnia/insomnia#5648]
### Expected Behavior

Prior to version 2022.7.0 users could select which GraphQL operation to send by simply placing the mouse cursor anywhere on the query in question. With version 2022.7.0 this functionality is no longer available. Now users have to select the query from a dropdown.

First, even though multiple bugs have been fixed in version 2022.7.1 the dropdown box still has problems. See attached image (this happens when going from multiple queries to one query without a name).
![operation-name](https://user-images.githubusercontent.com/6221648/211699857-151443bb-40b9-426b-b941-056e5f0c5cdc.JPG)

But second, more importantly, this is a huge regression, or at least a huge lack of functionality that existed prior. Consider that usually the mouse is already on the query the user is editing. Thus sending the query used to be just one click away. Now it usually takes me 3 or 4 clicks.

While this may seem like a trivial change at first, for those of us who work with a document containing dozens of queries it's a blocker. I had to downgrade to version 2022.6.0.

Some of this is of course muscle memory from previous functionality but consider:
1) If you have many similar named queries, it takes much longer to correlate the queries with the names in the dropdown.
2) If you have many queries with similar functionality, and you chose the neighboring query from the dropdown menu, you may not notice that the results are wrong.
3) Muscle memory: I loved that Insomnia was the only client that had this ability, and now it's hard to go without it.

### Actual Behavior

Users have to select the specific query they have to run from the Operations drop-down menu.

### Is there an existing issue for this?

- [X] I have searched the [issue tracker](https://www.github.com/Kong/insomnia/issues) for this problem.

### Additional Information

There was an issue raised for this: https://github.com/Kong/insomnia/issues/5631 but it is now closed. However this issue was related to the drop-down menu not working correctly, rather than bringing back the old functionality.

### Insomnia Version

2022.7.1

### What operating system are you using?

Windows

### Operating System Version

Windows 10

### Installation method

Download from the internet

### Last Known Working Insomnia version

2022.6.0

**Comment by itsfabijano** [comment]
+1

**Comment by goofy78270** [comment]
![image](https://user-images.githubusercontent.com/23242835/213031084-d2a05ae9-a882-48b1-a534-f2e4e8018e19.png)
need to add a space between operation name and "("

**Comment by gyles** [comment]
+1. #5631 made graphql barely useable again but having this functionality back is the ideal state.

**Comment by zleonov** [comment]
Yeah, if a dropdown is required both functionalities can be implemented. The dropdown menu could dynamically switch to the query the mouse cursor is on. But it's really not clear to me that a dropdown is required at all. The old functionality worked perfectly.

@filfreire This is such an important usability issue. Def. need to get this back. Let me us know if we can provide more info or confirm something.

**Comment by zleonov** [comment]
Hi @filfreire and team. I know you guys are busy, but I am wondering if you could provide any update on this issue? I do see that it has been verified. So does that mean it's been assigned some priority? Is this on the queue?

I love insomnia, so my team and I are sticking to version 2022.6.0 (the last version where this functionality worked), but I would love to tell them to turn back on auto-updates.

**Comment by vincenzodnp** [comment]
It would be grand to have this functionality back.
It's super faster, when you have multiple queries/mutations on the same page, to run them.
